tests/integration/commands/dev/dev.exec.test.js

```diff
@@ -5,7 +5,7 @@ import { test } from 'vitest'
 import { callCli } from '../../utils/call-cli.js'
 import { getCLIOptions, withMockApi } from '../../utils/mock-api.js'
 import { withSiteBuilder } from '../../utils/site-builder.ts'
-import routes from '../env/api-routes.ts'
+import { routes } from '../env/api-routes.ts'
```

Comment:
Is this import used anywhere else?

Reply:
It is used for all of the `env` test files.